common/test/acceptance/pages/lms/teams.py

```diff
@@ -281,6 +281,65 @@ def cancel_team(self):
         self.q(css='.create-team .action-cancel').first.click()
         self.wait_for_ajax()
 
+    def click_membership_button(self):
+        """Clicks the 'edit membership' button"""
+        self.q(css='.action-edit-members').first.click()
+        self.wait_for_ajax()
+
+    @property
+    def membership_button_present(self):
+        return self.q(css='.action-edit-members').present
+    
+    
+
+class EditMembershipPage(CoursePage):
+    """
+    Staff or discussion-privileged user page to remove troublesome or inactive
+    students from a team
+    """
+    def __init__(self, browser, course_id, team):
+        """
+        Set up `self.url_path` on instantiation, since it dynamically
+        reflects the current team.
+        """
+        super(EditMembershipPage, self).__init__(browser, course_id)
+        self.team = team
+        self.url_path = "teams/#teams/{topic_id}/{team_id}".format(
+            topic_id=self.team['topic_id'], team_id=self.team['id']
+        )
+
+    def is_browser_on_page(self):
+        """Check if we're on the team membership page for a particular team."""
+        self.wait_for_ajax()
+        if not self.url.endswith(self.url_path):
+            return False
+        if self.q(css='.edit-members').present:
+            return True
+        empty_query = self.q(css='.teams-main>.page-content>p').first
+        return (
+            len(empty_query.results) > 0 and
+            empty_query[0].text == "This team does not have any members."
+        )
```

Comment:
Is there a cleaner way I could do this last condition?

"Find the `p` element that's a child of a `.page-content` element that's a child of `.teams-main`. Count the number of results, if there's more than zero check the first one to compare it's text against this string."